src/main/resources/assets/securitycraft/lang/de_de.json

```diff
@@ -424,7 +424,7 @@
     "help.securitycraft.whitelist_module.info": "Das Whitelist-Modul verschont Spieler von einer Passworteingabe oder anderen Eigenschaften des Blocks.",
     "help.securitycraft.blacklist_module.info": "Das Blacklist-Modul ist das genaue Gegenteil zum Whitelist-Modul, z.B. blockiert es Spieler von der Eingabe eines Passwortes.",
     "help.securitycraft.harming_module.info": "Das Schadensmodul schadet Spielern, die nicht auf dem Whitelist-Modul sind, wenn diese z.B. durch einen Laser laufen.",
-    "help.securitycraft.smart_module.info": "Das schlaue Modul berücksichtigt auch Enchantments und Metadata eines Items z.B. im Inventarscanner.",
+    "help.securitycraft.smart_module.info": "Das schlaue Modul berücksichtigt auch Verzauberungs- und Metadata eines Items z.B. im Inventarscanner.",
```

Comment:
Reading over this a second time, changing it to Verzauberungs- und NBT-Daten would be best.